src/addin/client-interfaces/addin-client-open-help-args.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Interface for launching the help tab from an add-in.
+ */
+export interface AddinClientOpenHelpArgs {
+
+  /**
+   * The name of the help key to be launched.
+   */
+  helpKey?: string;
```

Comment:
also - need to document this in the readme

Reply:
updated. Also fixed above helpKey.